.shopify-build/polaris-icons-deploy-ruby.yml

```
@@ -1,12 +1,21 @@
 containers:
-  packagecloud:
-    docker: gcr.io/shopify-docker-images/ci/packagecloud:0.1
   ruby:
     docker: circleci/ruby:2.5.1-node
+  macos:
+    anka: macos-xcode
 
-shared:
-  build-gem: &build-gem
-    label: Build Gem
+steps:
+  - label: Run tests
+    container: ruby
+    timeout: 5m
+    run:
+      - cd packages-ruby/polaris_icons
+      - bundle install
```

```suggestion
      - - bundle: ~
```

https://shopify-build.docs.shopify.io/jobs/defining_jobs/#bundler